src/platforms/dialogflow/DialogflowEvent.ts

```diff
@@ -95,26 +86,20 @@ export class DialogflowEvent extends VoxaEvent {
    */
   protected initUser(): void {
     const { originalDetectIntentRequest } = this.rawEvent;
-    const { conv } = this.google;
+    const { conv } = this.dialogflow;
     const storage = conv.user.storage as any;
     let userId: string = "";
 
-    this.source = _.get(originalDetectIntentRequest, "source") || "google";
-
-    if (this.source === "google") {
-      if (conv.user.id) {
-        userId = conv.user.id;
-      } else if (_.get(storage, "voxa.userId")) {
-        userId = storage.voxa.userId;
-      } else {
-        userId = v1();
-      }
-
-      _.set(this.google.conv.user.storage, "voxa.userId", userId);
-    } else if (this.source === "facebook") {
-      userId = _.get(originalDetectIntentRequest, "payload.data.sender.id");
+    if (conv.user.id) {
+      userId = conv.user.id;
+    } else if (_.get(storage, "voxa.userId")) {
```

Comment:
i believe the storage is something that's only available on the google actions platform, and so is most of the content in the `conv.user`  object :thinking:

Reply:
Yes, that's why I'm overwriting this method in the `FacebookEvent` class